include/singa/io/reader.h

```diff
@@ -93,6 +95,29 @@ class BinFileReader : public Reader {
   const char kMagicWord[2] = {'s', 'g'};
 };
 
+/// TextFileReader reads tuples from CSV file.
+class TextFileReader : public Reader {
+ public:
+  ~TextFileReader() { Close(); }
+  /// \copydoc Open(const std::string& path)
+  bool Open(const std::string& path, int capacity = 10485760) override;
```

Comment:
capacity is not necessary here.
maybe remove the capacity argument.
for BinFileReader, add a 
```
set_buffer_capacity(size_t capacity)
```